components/x-privacy-manager/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@financial-times/x-privacy-manager",
-  "version": "1.1.0",
+  "version": "1.2.2",
```

Comment:
you don't need to do this, it's handled by the release process. most other packages have the version set to `0.0.0`.